start.py

```diff
@@ -9,6 +10,7 @@
 from osfoffline.gui.qt import OSFOfflineQT
 from osfoffline.utils.singleton import SingleInstance
 
+signal.signal(signal.SIGINT, signal.SIG_DFL)
```

Comment:
IIRC this is a "dirty" shutdown.
You may want to disable this when not in debug mode.